pandas/compat/pickle_compat.py

```diff
@@ -50,26 +52,31 @@ def load_reduce(self):
             print(func, args)
         raise
 
-    stack[-1] = value
```

Comment:
Honestly, I have no idea.  You should ask yourself that :smile: (your name shows up when look at the `blame` for this file)